src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -64,28 +71,32 @@ class JenkinsJobManager {
 
         for(ConcreteJob missingJob in missingJobs) {
             println "Creating missing job: ${missingJob.jobName} from ${missingJob.templateJob.jobName}"
+            try {
+                // This creates missing jobs using the missingJob name. Note that templateJobs is a List.
+                jenkinsApi.cloneJobForBranch(missingJob, templateJobs)
 
-            // This creates missing jobs using the missingJob name. Note that templateJobs is a List.
-            jenkinsApi.cloneJobForBranch(missingJob, templateJobs)
-
-            // Rev.com-branch is -DtemplateJobPrefix
-            // Rev.com-branch-deploy- is the Jenkins job template
-            // Rev.com-branch-build- is the Jenkins job template
+                // Rev.com-branch is -DtemplateJobPrefix
+                // Rev.com-branch-deploy- is the Jenkins job template
+                // Rev.com-branch-build- is the Jenkins job template
 
-            if (missingJob.jobName.contains("Rev.com-build-unittest")) {
-                // If the job contains unittest enable it. Jobs start out disabled so -deploy- will remain disabled.
-                jenkinsApi.enableJob(missingJob.jobName)
-            }
-            if (!missingJob.jobName.contains("Rev.com-deploy-unittest")) {
-                // Prevent double enabling of unittest jobs.
-                if (!missingJob.jobName.contains("Rev.com-build-unittest")) {
-                    // If the job doesn't contain unittest enable it. Jobs start out disabled.
+                if (missingJob.jobName.contains("Rev.com-build-unittest")) {
+                    // If the job contains unittest enable it. Jobs start out disabled so -deploy- will remain disabled.
                     jenkinsApi.enableJob(missingJob.jobName)
                 }
+                if (!missingJob.jobName.contains("Rev.com-deploy-unittest")) {
+                    // Prevent double enabling of unittest jobs.
+                    if (!missingJob.jobName.contains("Rev.com-build-unittest")) {
+                        // If the job doesn't contain unittest enable it. Jobs start out disabled.
+                        jenkinsApi.enableJob(missingJob.jobName)
+                    }
+                }
+                if (startOnCreate && missingJob.jobName.contains("Rev.com-build-")) {
+                    // Starting -build- has the downstream job of -deploy- when successful.
+                    jenkinsApi.startJob(missingJob)
+                }
             }
-            if (startOnCreate && missingJob.jobName.contains("Rev.com-build-")) {
-                // Starting -build- has the downstream job of -deploy- when successful.
-                jenkinsApi.startJob(missingJob)
+            catch(Exception ex) {
+                failed = true
```

Comment:
Creating jobs does not fail very often. I am not sure we need this here. But I guess it doesn't hurt.

We should add some logging here as well.